docs/paper/dev/api/scheduler.md

```diff
@@ -0,0 +1,54 @@
+﻿---
+slug: /dev/scheduler
+---
+
+# Scheduler
+
+The `BukkitScheduler` can be used to schedule your code to be run later or run it repeatedly.
+
+## What is a tick?
+
+Every game runs something called a game loop which essentially executes all the logic of the game over and over,
+a single execution of that loop is called a 'tick'.
+
+In Minecraft's case the amount of ticks per second is 20, meaning that the game loop is executed 20 times per second.
+With some math we can see that 1 tick is equal to 50 milliseconds. A tick taking more than 50ms to execute is the moment
+when your server starts to fall behind on its work and lag.
+
+### Converting between SI units and Minecraft ticks
+
+Every method of the scheduler that takes a delay or period uses ticks as a unit of time.
+
+:::warning
+
+The above also applies to the asynchronous methods. If you need a scheduler with millisecond precision, create your own
+[`ScheduledExecutorService`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/concurrent/ScheduledExecutorService.html).
+
+:::
+
+Converting from human units to ticks and back is as simple as:  
+`ticks = seconds * 20`  
+`seconds = ticks / 20`
+
+:::note
+
+In Java and Kotlin to not lose precision you must use `ticks / 20.0`.
```

Comment:
I don’t think this is the place to explain basic java

Reply:
considering the IDE does display a warning when the variable's type is `float`/`double` and above is pseudo-code, agreed